nixos/modules/sysvm.nix

```diff
@@ -82,12 +82,12 @@ in {
       };
     };
 
-    wifiManager = mkOption {
+    wifiIface = mkOption {
```

Comment:
Why do you change this option into a string?

Reply:
To get the name of the correct wifi interface.

Reply:
This is the option to turn on the wifi manager service, which should only be present in the netvm agent. If you want an option to pass the wifi interface name please make an additional option for it. Also, if we keep the hwid as generic service, we need an additional switch so one can enable it in a particular instance of the agent. 

Currently, the implementation only enables this service if the wifi manager is enabled. So if you want to make this generic we should have a clean separation, i.e.

1. a boolean switch to turn this service on (off by default)
2. a string parameter for the interface name